source/_posts/2018-01-03-using-git-hooks-to-improve-your-day-to-day-workflow.markdown

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: "Using git hooks to improve your day-to-day workflow"
+category: git
+comments: true
+author:
+  name: Federico Kauffman
+  email: federico@wyeworks.com
+  twitter_handle: fedekauffman
+  github_handle: fedekau
+  image: /images/team/federico-kauffman.jpg
+  description: Software Engineer at Wyeworks. Currently working with Javascript and Ruby. Learnaholic.
+---
+
+If you have been developing software for some time you have probably noticed that there are lot of things that can go wrong, no matter how hard you try there is always something you might forget because after all, we are just humans doing an extremly difficult task: Telling a computer what to do.
+
+In this post I will expose some of the **git-hooks** we use in some projects here at Wyeworks to make developers life easier by preventing bad commits to even leave their computers. I will cover *linting*, *tests* and *commit formating* use cases.
+
+<!--more-->
+
+## What are git-hooks?
+
+Like many other Version Control Systems, Git has a way to fire off custom scripts when certain important actions occur. There are two groups of these hooks: *client-side* and *server-side*. Client-side hooks are triggered by operations such as committing and merging, while server-side hooks run on network operations such as receiving pushed commits. You can use these hooks for all sorts of reasons.
+
+## Installing a Hook
+
+The hooks are all stored in the hooks subdirectory of the Git directory. In most projects, that’s `.git/hooks`. When you initialize a new repository with `git init`, Git populates the hooks directory with a bunch of example scripts, many of which are useful by themselves; but they also document the input values of each script. All the examples are written as shell scripts, with some Perl thrown in, but any properly named executable scripts will work fine – you can write them in Ruby or Python or whatever language you are familiar with. If you want to use the bundled hook scripts, you’ll have to rename them; their file names all end with `.sample`.
```

Comment:
I would re-write the last sentence to something like this

> If you want to re-use one of the existing bundled hook scripts you'll have to rename it, since they all contain `.sample` at the end.